src/server/router.js

```diff
@@ -0,0 +1,24 @@
+
+const handlers = require('./handlers');
+
+
+const router = (request, response) => {
+  if (request.method === 'GET') {
+    if (request.url === '/') {
+      handlers.homeHandler(request, response);
+    } else if (request.url.includes('/search?value=')) {
+      handlers.searchMoviesHandler(request, response);
+    } else if (request.url.includes('/getMovies?id=')) {
+      handlers.getMoviesHandler(request, response);
+    }else if (request.url.includes('/PostMovies?id=')) {
```

Comment:
```/PostMovies``` is a ```POST``` method not ```GET``` method, it's not right to put it inside this condition 
```js
 if (request.method === 'GET') { 
``` 
You should make another condition for ```POST``` methods